console/jsre/deps/web3.js

```diff
@@ -6166,6 +6524,58 @@ var klay = function () {
     ];
 };
 
+
+/// @returns an array of objects describing web3.eth.filter api methods
+var eth = function () {
+	var newFilterCall = function (args) {
```

Comment:
Another wrong indentation ;)

Reply:
My bad, thank you for detail review; i modified indentation again!